ci_group/pyproject.toml

```diff
@@ -20,13 +20,19 @@ classifiers = [
     "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
 ]
 packages = [{ include = "revolve2" }]
+include = [{ path = "revolve2/ci_group/morphological_novelty_metric/*.so", format = "wheel" }]
```

Comment:
I am quite sure this will break on windows, as the extension for libraries is different

Reply:
it did not :D